js/generate-index.js

```diff
@@ -12,7 +10,16 @@ let templateXhr = new XMLHttpRequest();
 templateXhr.responseType = 'text';
 templateXhr.open('GET', '../js/template-index');
 templateXhr.onload = function() {
+  // Save useful content from initial document
+  // (custom elements in the head, main header)
```

Comment:
The "custom elements" comment here is a bit confusing (with the terminology in the [HTML Standard](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements)).

Although I understand what it means in this context, future readers (including me, perhaps) might be confused.